capstone/src/main/java/com/google/sps/data/SurveyResponse.java

```diff
@@ -0,0 +1,68 @@
+package com.google.sps.data;
+
+/* This object represents each response a user submits */
+public class SurveyResponse {
+    private String id;
+    private double score;
+    private String gender;
+    private String ageRange;
+    private long responseTime;
+    private String date; 
+    private long day;
```

Comment:
WHat's the difference between these two?

Reply:
Date comes from the surveys API and day indicates how many days have passed since the surveys opened up. I kept this field here just to load the charts initially, I will be changing them to work with the Surveys API for the MVP

Reply:
Hmm instead of having a field to store the number of days have passed, I'd recommend storing a Survey `start_date` instead.

The reason being is that you can calculate the days passed based on start date quite easily. You can also do more interesting things with the start_date.